### DISCUSSION TITLE
Meetup 2023 (10k event)

### DISCUSSION THREAD

**Discussion by frosch123** [OpenTTD/OpenTTD#10002] [General]
## What
Visit a railway museum and chat about trains/buses/OpenTTD, maybe eat some food.

## Where
[Train World](https://www.trainworld.be/en) in Brussels.

## When
June 10, 2023, at 13:00. We will meet directly outside the main entrance to the museum. Name tags will be provided -- look for the group wearing them!

Lunch beforehand and dinner afterwards are likely, but have not been planned yet. Anyone have recommendations?

## Who can attend
Anyone who can take care of themselves.
* You must be an adult (18+).
* You must not get so drunk, that you cause trouble.

## Who to expect
Low two-digit number of people.
* Computer nerds.
* Several developers, some aged 30 to 50. (warning: people may tell rose-tinted stories from their youth)

You are expected to pay for your own travel, food, and museum admission.

(edited by @2TallTyler)

**Comment by jdopss** [comment]
Miniatur Wunderland is great, but some people could have trouble (ticket availability/prices) so i just wanted to note that

**Reply by James103** [reply]
I just checked on the website, and tickets for a group of 14 (the max allowed at once by the system) are sold out through at least this Halloween. In order to make it in, you will need to book at least 70-80 (and ideally a hundred) days in advance.

You might want to search for other model railways and railway museums in Europe and/or the UK (keeping in mind that Transport Tycoon, the game that eventually led to OpenTTD, was made in Scotland).

**Reply by frosch123** [reply]
Yeah, looks like it is not really an option.

**Comment by nielsmh** [comment]
For me: Don't make it too early, need to be able to get travel tickets and plan a bit ahead, and possibly get half/whole days off from work. Apart from that, mostly any larger city in Denmark, Germany, the Netherlands, the UK, or southern half of Sweden would be okay with me.
A Saturday afternoon meetup time is probably the easiest to plan for.

**Comment by planetmaker** [comment]
One month ahead is kinda short-notice,but if it fits my already somewhat filled calendar (half the weekends somewhat booked) it would be a welcome weekend activity for me :)

**Comment by Quast65** [comment]
Has this ever been considered as a location?
https://www.madurodam.nl/
Next to The Hague in The Netherlands (and with The Netherlands being so small, also close to Rotterdam or Amsterdam ;-) )

**Reply by Kuhnovic** [reply]
Madurodam is awesome. Another options in NL would be the "Spoorwegmuseum", the Dutch Railway Museum in Utrecht.

**Comment by 2TallTyler** [comment]
It's almost next summer, so let's get this going!

I'd like to organize a meetup on either June 10, 11, 17, or 18, at [Train World](https://www.trainworld.be/en), in Brussels.

My hope is that Brussels is easy enough to get to from both the continent and the UK. And yes, it's short notice, but I'll be in Germany for work and couldn't attend any other meetup...and somebody has to take initiative to make this happen!

Who would be interested in attending, and on which date?

**Reply by nielsmh** [reply]
Either of the dates could probably work for me, but the Saturdays are definitely easier, in a large part from the question of _getting home again_.

Also, what time of day would the gathering be? From early morning? Around lunchtime? Evening?

**Reply by Eddi-z** [reply]
I probably couldn't make either of those weekends...

**Reply by 2TallTyler** [reply]
I plan to arrive mid-morning and leave after dinner, so whenever works for people, but I imagine the main meetup would be lunchtime and early afternoon.

**Reply by TrueBrain** [reply]
I hear lunch + dinner :P

I can make any of those dates work; will go by car, so any Dutchy who wants to tag along, let me know :)

**Reply by michicc** [reply]
Purely from a travel logistics point, Saturday seems easier to do. Not totally sure I could attend, but both weekends are equally likely for me.

**Reply by nielsmh** [reply]
Regardless of which date, my plan would probably involve going by air, and having an overnight hotel stay. Sadly, getting to Brussels by train from Denmark is an ordeal, typically involving a lot of changes and long waits, often in the middle of the night too.
In the interest of more flexibility in ordering tickets etc. I think Saturday 17th would be my preferred date, closely followed by Sunday 18th.

**Reply by frosch123** [reply]
Both dates work for me. I would probably stay a few days more before/after the event.

**Reply by Eddi-z** [reply]
i could maybe make room in the weekend of 10/11. but a requirement would be that someone organizes something to do in the evening and a location to spend the night (doesn't need to be a hotel, could also sleep at someone's place)

**Reply by Kuhnovic** [reply]
10/11 works for me. I will be in belgium the other weekend, but i'll be rotting away at a festival.

@TrueBrain Can I reserve a seat 😉?

**Reply by rubidium42** [reply]
I think I can make those days work except for the 17th, unless that ends early/mid afternoon.

**Reply by 2TallTyler** [reply]
Let's do June 10, then. I'll work on finalizing details.

**Reply by orudge** [reply]
Unlikely to be able to make it without spending a small fortune it seems, but maybe can try to arrange something for the 20th anniversary in (or around) March next year…

**Reply by FLHerne** [reply]
Similar, I can manage the dates but not at a cost I can justify from the UK (except Flixbus with unreasonable travel times instead).

If a couple more UK people were going perhaps we could arrange a carpool and split the cost of fuel/ferry/Tunnel? I don't have a car though - only a narrowboat, and that isn't a viable transport option to Brussels!

**Reply by LordAro** [reply]
Moi aussi

**Reply by TrueBrain** [reply]
@Kuhnovic 

> 10/11 works for me. I will be in belgium the other weekend, but i'll be rotting away at a festival.
> 
> @TrueBrain Can I reserve a seat 😉?

Please drop me an email at truebrain at openttd dot org so we can talk details :)

**Comment by Eddi-z** [comment]
So, how do we commucate actually meeting up? I won't have internet during travel.

**Reply by 2TallTyler** [reply]
We will meet outside the main entrance of the museum at 13:00. I'll bring name tags, so look for the group of people wearing them.

That said, who is getting there early enough to grab lunch? I get in around 9:30 am. Shall we say, anyone who wants to do lunch should meet outside the main entrance at 11:30? 

@TrueBrain, what time is your carpool arriving?

Edit: I won't have internet either, so ping somebody else if you have trouble. 😛

**Reply by Eddi-z** [reply]
My train will be arriving at either 9:49, 10:49 or 11:49 at bussels nord (depending on personal mood at the day)

**Reply by TrueBrain** [reply]
> 
> @TrueBrain, what time is your carpool arriving?

Before lunch!

And most EU people with a mobile dataplan will have Internet connection, so we should be good I guess. Maybe a special Discord room so we don't annoy the others might be useful :)

**Reply by Eddi-z** [reply]
... if i had that, i wouldn't say that i don't have that...

**Reply by michicc** [reply]
I should be in Brussels proper at 9:00 or so (depending on the exact location of course).

So elevenses before lunch somewhere? 😁

**Reply by 2TallTyler** [reply]
Sure, pick a time and place to meet (in front of the museum to keep it easy?) and I'll be there.

**Reply by Eddi-z** [reply]
i was thinking if i take the early train, to do some generic sightseeing first...

**Reply by nielsmh** [reply]
My bus is scheduled to arrive at Brussels Nord station at 11:30 (and I do have a data plan.)
I would like to join for lunch if possible, but probably wouldn't be able to make an 11:30 meetup for that.

**Reply by michicc** [reply]
I'm so not familiar with Brussels, but looking at Google Maps, I don't see that much restaurants/cafes directly around Train World. Nord station seems to have a bit more around, but it looks like a lot of offices there, so might not be that busy/lively on a Saturday.

**Reply by nielsmh** [reply]
As a general precaution: Remember to bring cash (EUR), you cannot be sure shops (especially eateries) will accept credit cards at all.

**Reply by michicc** [reply]
@2TallTyler Unless somebody wants to do something earlier, I guess I'll be outside Train World at 11:00 (and either wait for someone or just scout the area a bit).

I am fully connected, so I will check back here and Discord for any earlier activities.

**Reply by 2TallTyler** [reply]
Great. Let's meet at 11 outside Train World and then find a place for elevenses/lunch. Then @michicc  can post here and on Discord where we've gone so @nielsmh and others can find us. 🙂 

For later arrivals or those who can't access data, we will still be meeting at 13:00 outside Train World for the museum visit. Dinner will be easier to organize since we'll all be in the same place.
